golem-rpc-api/src/net.rs

```diff
@@ -132,13 +132,14 @@ pub struct AclStatus<Identity> {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
-pub struct AclRuleItem<Identity>(
-    pub Identity,
-    pub AclRule,
+pub struct AclRuleItem<Identity> {
```

Comment:
I do not like this. AclRuleItem have the same structure for node_id acl and ip acl. If you change this 
how would look like response for `net.peer.acl_ip` ? Now is:

```json
{"default_rule": "allow", "rules": [["127.0.0.1", "deny", nil]]}
```

After your change it will be:

```
{"default_rule": "allow", "rules": [{"node_id": "127.0.0.1", "node_name": "????", "rule":  "deny", "deadline": nil]]}
```

Reply:
Variable names were improperly used (e.g. node_id for node_id/ip), I couldn't realise use of `ip` case in first glance. But then I extended `node_name` for `ip` as well. 

https://github.com/golemfactory/golem/pull/4701/files#diff-77a5077d78c46cf901081cf89bac9d34R355

https://github.com/golemfactory/golem/pull/4701/files#diff-91438fae6ece17120b4bbea18c01d9e5R279

I know tuple usage is much more easier but I had to create something tells what value is what for.  Maybe I should rename  `node_id` key as `identity`, is that okay?